app/configurator/components/icon-button.tsx

```diff
@@ -0,0 +1,82 @@
+import { ButtonBase, Theme, Typography } from "@mui/material";
+import { makeStyles } from "@mui/styles";
+import clsx from "clsx";
+
+import { getFieldLabel } from "@/configurator/components/field-i18n";
+import { getIconName } from "@/configurator/components/ui-helpers";
+import { FieldProps } from "@/configurator/config-form";
+import { Icon } from "@/icons";
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+    alignItems: "center",
+    width: 91,
+    height: 64,
+    borderRadius: 8,
+    transition: theme.transitions.create(["background-color", "color"], {
+      duration: theme.transitions.duration.shorter,
+    }),
```

Comment:
TIL theme transitions. I liked your way of using shorthands in swiss federal ci. I now use 

```
({ palette: p, spacing: s, shadows: e, }) => {}
```

Maybe transitions could `t` :)